packages/migrations/src/createStakerNetworkAndConnectStakerPkgs.ts

```diff
@@ -0,0 +1,42 @@
+import { docker } from "@dappnode/dockerapi";
+import { params } from "@dappnode/params";
+import { logs } from "@dappnode/logger";
+import { ensureStakerPkgsNetworkConfig } from "@dappnode/stakers";
+import { networks } from "@dappnode/types";
+
+/**
+ * Creates the staker network and connects the staker packages to it
+ */
+export async function createStakerNetworkAndConnectStakerPkgs(): Promise<void> {
+  await createDockerStakerNetwork();
+  for (const network of networks) await ensureStakerPkgsNetworkConfig(network);
+}
+
+/**
+ * Creates the docker staker network
+ */
+async function createDockerStakerNetwork(): Promise<void> {
+  try {
+    const stakerNetwork = docker.getNetwork(params.DOCKER_STAKER_NETWORK_NAME);
+    await stakerNetwork.inspect();
```

Comment:
Why calling here inspect?

Reply:
Is it necessary for the error to be thrown?

Reply:
Can't you just check if the network exists and then create it if not? Instead of triggering an error for something that is expected